WiseUnpacker/SabreTools.Serialization.Wrappers/WiseSectionHeader.cs

```diff
@@ -214,49 +218,197 @@ private bool ExtractHeaderDefinedFiles(Stream data, string outputDirectory, bool
 
             // Does output size include the crc32? Doesn't seem to?
 
-            // Extract main MSI file
-            var expected = new DeflateInfo
-            {
-                InputSize = MsiFileEntryLength,
-                OutputSize = MsiFileEntryLength - 4,
-                Crc32 = 0
-            };
-            data.Seek(data.Position - MsiFileEntryLength, 0);
-            if (InflateWrapper.ExtractFile(data, "ExtractedMsi.msi", outputDirectory, expected, false, includeDebug)
-                == ExtractionStatus.FAIL)
+            // Comparing against extractionstatus GOOD for now.
+            
+            // Extract first executable, if it exists
+            if (ExtractFile(data, "FirstExecutable.exe", outputDirectory, FirstExecutableFileEntryLength, includeDebug)
+                != ExtractionStatus.GOOD)
```

Comment:
Please look at awkward line wrapping. In all places that `ExtractFile` is currently called, it is a single line. Yeah, not the nicest to read, but please try to maintain consistency. This applies to all below.